### PR TITLE
Update OpenAPI spec and fix changes.

### PR DESCRIPTION
**Pull request by tasn**
This also includes fixes to the Python API that were broken regardless.
This includes a few small API breaks that are tolerable for now. It's one
of the last cleanups before we don't break API anymore.

### CODE REVIEW

**Comment by tasn** [comment]
@svix-frank, please make sure that the Go API (and tbh, the rest of the APIs) are correct.

**Comment by tasn** [comment]
@svix-frank, can you please review it to make sure I haven't messed up Python and JS too so I can merge and release a new version? Thanks!

File: python/svix/api.py [hunk 1]
```diff
@@ -120,7 +132,7 @@ def delete(self, app_id: str, endpoint_id: str) -> None:
                 app_id=app_id, endpoint_id=endpoint_id
             )
 
-    def get_secret(self, app_id: str, endpoint_id: str) -> EndpointOut:
+    def get_secret(self, app_id: str, endpoint_id: str) -> EndpointSecretOut:
```
Comment (svix-frank) [inline python/svix/api.py:135]: Nice catch

**Review by svix-frank — COMMENTED** [review]
LGTM sorry for the delay

**Review by svix-frank — APPROVED** [review]

**Pull request merged**